card/mod/05_email/set/all/follow.rb

```diff
@@ -137,37 +134,35 @@ def follow_rule_applies? follower_id
       )
 
       applicable =
-        if test = FollowOption.test[ value_code ]
+        if (test = FollowOption.test[value_code])
           test.call follower_id, accounted_ids
         else
           accounted_ids.include? follower_id
         end
 
-      return value.gsub( /[\[\]]/, '' ) if applicable
+      return value.gsub(/[\[\]]/, '') if applicable
     end
   end
-  return false
+  false
 end
 
-
 def with_follower_candidate_ids
   @follower_candidate_ids = {}
   yield
   @follower_candidate_ids = nil
 end
 
-
 # the set card to be followed if you want to follow changes of card
 def default_follow_set_card
   Card.fetch("#{name}+*self")
 end
 
-
 # returns true if according to the follow_field_rule followers of self also
 # follow changes of field_card
 def followed_field? field_card
-  (follow_field_rule = rule_card(:follow_fields)) || follow_field_rule.item_names.find do |item|
-     item.to_name.key == field_card.key ||  (item.to_name.key == Card[:includes].key && included_card_ids.include?(field_card.id) )
+  (follow_field_rule = rule_card(:follow_fields)) ||
+    follow_field_rule.item_names.find do |item|
+     item.to_name.key == field_card.key || (item.to_name.key == Card[:includes].key && included_card_ids.include?(field_card.id) )
```

Comment:
Line is too long. [130/80]<br>Use 2 (not 3) spaces for indentation.<br>Space inside parentheses detected.